pages/content/amp-dev/documentation/courses/beginning-course/our-first-amp-page.md

```diff
@@ -350,7 +350,7 @@ The following tags are required in AMP pages. Valid AMP pages must:
 
 - Contain a `<meta charset="utf-8">` tag as the first child of their `<head>` tag.
 
-- Contain a `<meta name="viewport" content="width=device-width"> **Note**: It's also recommended to include `initial-scale=1`.
+- Contain a `<meta name="viewport" content="width=device-width">` **Note**: It's also recommended to include `initial-scale=1`.
```

Comment:
I completely forgot to discuss it, but did you see the chrome 91 change? https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_05_fid.md